.gitmodules

```
@@ -81,4 +81,9 @@
 [submodule "externals/ffmpeg-core"]
 	path = externals/ffmpeg-core
 	url = https://github.com/shadps4-emu/ext-ffmpeg-core.git
-	shallow = true
\ No newline at end of file
+	shallow = true
+[submodule "externals/dear_imgui"]
+	path = externals/dear_imgui
+	url = git@github.com:shadps4-emu/ext-imgui.git
```

all neighboring urls are `https`. 
```suggestion
	url = https://github.com/shadps4-emu/ext-imgui.git
```